students/BrandonHenson/Lesson9/Mailroom_5.py

```diff
@@ -0,0 +1,138 @@
+# Brandon Henson
+# 5/17/18
+# Lesson 09 mailroom_5.py
+# Refactoring
+
+
+class Donor(object):
+
+    donor_history = {'Brandon Henson': [1005.49, 3116.72, 5200],
+                        'Alicia Henson': [21.47, 1500],
+                        'Michael Green': [2400.54],
+                        'Brandon Henson Jr': [355.42, 579.31],
+                        'Kaiya Henson': [636.9, 850.13, 125.23]}
+
+    name_list = [i for i in donor_history]
+
+    def __init__(self, name, *donation):
+        self.donations = []
+        self.name = name
+        self.donation = donation
+        for i in self.donation:
+            self.donations.append(i)
+        Donor.name_list.append(self.name)
+        Donor.donor_history[self.name] = self.donation
+
+    @property
+    def total(self):
+        return sum(self.donations)
+
+    @property
+    def average(self):
+        return self.total/self.num_of_donations
+
+    @property
+    def num_of_donations(self):
+        return len(self.donations)
```

Comment:
These would be great use cases for properties on the donor collection class, but you don't appear to be using them (or the `donations` instance attribute) anywhere in your code as this is currently implemented